docs/pages/workflows/variables.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Variables
+description: Learn how to use variables in Workflows.
+---
+
+You can use variables in your workflows to pass data between jobs or steps. Variables are useful for conditionals on subsequent jobs or afftecting the output of a workflow.
+
+You can set variables in the workflow file with the `set-output` function:
+
+```yaml .eas/workflows/variables.yaml
+jobs:
+  fingerprint:
+    outputs:
+      # @info Sets the `fingerprint_hash_for_workflow` as a variable #
+      fingerprint_hash_for_workflow: ${{ steps.fingerprint_step_id.outputs.fingerprint_hash }}
+      # @end #
+    steps:
+      - name: Set fingerprint hash
+        id: fingerprint_step_id
+        # @info Exposes the `fingerprint_hash` variable to other steps in this job #
+        outputs: [fingerprint_hash]
+        # @end #
+        run: |
+          FINGERPRINT_HASH=$(npx expo-updates fingerprint:generate --platform ios | jq '.hash')
+          # @info Sets the `fingerprint_hash` as a variable #
+          set-output fingerprint_hash $FINGERPRINT_HASH
+          # @end #
+      - name: Do something with the fingerprint hash
+        run: |
+          # @info Accesses the `fingerprint_hash` variable #
+          echo ${{ steps.fingerprint_step_id.outputs.fingerprint_hash }}
+          # @end #
+    maybe_build:
+      needs: [fingerprint]
+      # @info Accesses the `fingerprint_hash_for_workflow` variable #
+      if: ${{ needs.fingerprint.outputs.fingerprint_hash_for_workflow != '1234' }}
+      # @end #
+      type: build
+      params:
+        platform: ios
+```
```

Comment:
````suggestion
```yaml .eas/workflows/variables.yaml
jobs:
  fingerprint:
    outputs:
      # @info Sets the <CODE>fingerprint_hash_for_workflow</CODE> as a variable #
      fingerprint_hash_for_workflow: ${{ steps.fingerprint_step_id.outputs.fingerprint_hash }}
      # @end #
    steps:
      - name: Set fingerprint hash
        id: fingerprint_step_id
        # @info Exposes the <CODE>fingerprint_hash</CODE> variable to other steps in this job #
        outputs: [fingerprint_hash]
        # @end #
        # @info Sets the <CODE>fingerprint_hash</CODE> as a variable #
        run: |
          FINGERPRINT_HASH=$(npx expo-updates fingerprint:generate --platform ios | jq '.hash')
          set-output fingerprint_hash $FINGERPRINT_HASH
        # @end #
      - name: Do something with the fingerprint hash
        # @info Accesses the <CODE>fingerprint_hash</CODE variable #
        run: |
          echo ${{ steps.fingerprint_step_id.outputs.fingerprint_hash }}
        # @end #
    maybe_build:
      needs: [fingerprint]
      # @info Accesses the <CODE>fingerprint_hash_for_workflow</CODE> variable #
      if: ${{ needs.fingerprint.outputs.fingerprint_hash_for_workflow != '1234' }}
      # @end #
      type: build
      params:
        platform: ios
```
````

The formatting of code comments for sequential jobs under run was broken:

![CleanShot 2024-11-16 at 22 26 25@2x](https://github.com/user-attachments/assets/176fbf28-8091-4df4-b0ca-80024efa89b9)


For now, I could only fix them if the comment surrounds the whole line (including the step keyword)

![CleanShot 2024-11-16 at 22 25 19@2x](https://github.com/user-attachments/assets/86171566-dd48-4648-980f-d809979ef28a)

Also, let's use `<CODE>` to highlight code syntax instead of back ticks since back ticks do not work in annotated comments to have the same affect

![CleanShot 2024-11-16 at 22 23 43@2x](https://github.com/user-attachments/assets/b29e4b4b-bbe3-4226-af10-b9dd6e02cf30)